sdk/core/Azure.Core/src/Shared/PrivilegedOperationAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Decorates a potentially privileged operation.
```

Comment:
What is the definition of "privileged" in this context?   If I'm a new library author and see this attribute included or used in the source for another product, how do I know whether or not I should use it, and what to apply it to?

It feels like we should have a link here to documentation that can answer those authoritatively.

Reply:
See the other comment - this is still in progress.